.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ workflows:
           matrix:
             parameters:
               # TODO: Revisit why pyenv doesn't recognize 3.12
-              python_version: ["3.8", "3.9", "3.10", "3.11"] # "3.12"
+              python_version: ["3.9", "3.10", "3.11"] # "3.12"
```

Comment:
On the topic of python versions, I've opened this: https://github.com/arangodb/python-arango/pull/362

Feel free to merge after this PR